core/src/main/java/nl/nn/adapterframework/webcontrol/pipes/ExecuteJdbcProperties.java

```diff
@@ -48,12 +49,12 @@ public void configure() throws ConfigurationException {
 	}
 
 	@Override
-	public Message doPipeWithTimeoutGuarded(Message input, IPipeLineSession session) throws PipeRunException {
+	public PipeRunResult doPipeWithTimeoutGuarded(Message input, IPipeLineSession session) throws PipeRunException {
 		String method = (String) session.get("method");
 		if (method.equalsIgnoreCase("GET")) {
-			return Message.asMessage(doGet(session));
+			return new PipeRunResult(getForward(), doGet(session));
 		} else if (method.equalsIgnoreCase("POST")) {
-			return Message.asMessage(doPost(session));
+			return new PipeRunResult(getForward(), doPost(session));
```

Comment:
En wat kan hier nou uit timen? En hoe wordt dat gekilled na een timeout?
Als dat is via de timeout van de FixedQuerySender, dan is de timeout van deze pipe niet nodig.